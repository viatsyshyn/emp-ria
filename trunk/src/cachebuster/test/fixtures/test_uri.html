<!DOCTYPE html>
<html>
<head>
    <title></title>
    <link href="test/fixtures/test.css?a=0" type="text/css" rel="stylesheet" />
    <script src="test/fixtures/test.js" type="text/javascript"></script>
    <link href="test/fixtures/favicon.ico" rel="shortcut icon" />
</head>
<body>
    <img src="test/fixtures/test.gif"/>
</body>
</html>