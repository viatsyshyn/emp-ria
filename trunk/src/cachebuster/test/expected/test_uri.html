<!DOCTYPE html>
<html>
<head>
    <title></title>
    <link href="test/fixtures/test.css?a=0&_=74kbgu" type="text/css" rel="stylesheet" />
    <script src="test/fixtures/test.js?9qyb4a" type="text/javascript"></script>
    <link href="test/fixtures/favicon.ico" rel="shortcut icon" />
</head>
<body>
    <img src="test/fixtures/test.gif?l2g13b"/>
</body>
</html>